static/js/main.js

```diff
@@ -75,6 +75,7 @@ var monito = (function (preferences, requests, gui) {
         init: init,
         refresh: autoRefresh,
         displayInfo: displayInfo,
-        displayMenu: displayMenu
+        displayMenu: displayMenu,
+        showSnackbar: _helpers.showSnackbar
     }
 })(_prefs, _requests, _gui);
```

Comment:
you would need to pass in the _helpers here as the last argument and then you'd be able to use it inside.